src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -8,20 +8,40 @@
 
 namespace System.Collections.Generic
 {
+    /// <summary>
+    /// A circular doubly linked list.
```

Comment:
> Yep, there's a plan in place

@cartermp, should we close this PR then and allow that process to happen, or should we clean up this PR and merge it?  I appreciate @SamuelEnglard's contributions, but if all of this will be done in a consistent fashion across the repo by the tool, I'd rather allow that process to take place and then manually clean up after it anything that needs to be fixed.